wallet/src/Pos/Wallet/Web/Pending/Worker.hs

```diff
@@ -125,7 +127,10 @@ processPtxs
     => SendActions m -> SlotId -> [PendingTx] -> m ()
 processPtxs sendActions curSlot ptxs = do
     mapM_ processPtxInNewestBlocks ptxs
-    processPtxsToResubmit sendActions curSlot ptxs
+
+    if walletTxCreationDisabled
+    then logDebug "Transaction resubmission is disabled"
+    else processPtxsToResubmit sendActions curSlot ptxs
```

Comment:
This also disables it for the re-submission worker. So it may not also be needed for `submitAndSavePtx`, but no harm.